packages/rke2-canal/package.yaml

```diff
@@ -1,2 +1,4 @@
 url: local
+# This repository does not use releaseCandidateVersions, so you can leave this as 00.
+releaseCandidateVersion: 00
```

Comment:
Why do you add this?

Reply:
hi @manuelbuil 
Thanks for the review.
It was included when I "forked" the file, so I left it in.
If it is not needed, we will delete it.

Reply:
AFAIK, this is not needed. However, in that same file, you should increase the packageVersion, otherwise CI will complain

Reply:
Thank you very much for your kindness.
I have deleted it.

Reply:
You forgot to increase the `packageVersion` in `packages/rke2-canal/package.yaml`. That's why CI is failing

Reply:
sorry for talomg up your time.
Updated 07 to 08.
Thanks for the review.

Reply:
thanks!